### ISSUE TITLE
ERROR: Could not find a version that satisfies the requirement tensorflow-gpu==1.12.0 (from versions: none)

### ISSUE THREAD

**Issue by magician11** [openai/gpt-2#139]
After running `pip3 install tensorflow-gpu==1.12.0`

I see

```
  ERROR: Could not find a version that satisfies the requirement tensorflow-gpu==1.12.0 (from versions: none)
ERROR: No matching distribution found for tensorflow-gpu==1.12.0
```

I'm running `Python 3.7.0` on a macOS Mojave.

I see [a closed issue for this repo that says only Python 3.6 is supported](https://github.com/openai/gpt-2/issues/6).

And I see [on the Tensorflow repo that Python 3.7 is supported on all operating systems](https://github.com/tensorflow/tensorflow/releases/tag/v1.13.1).

So that's a bit confusing.

If I have to use Python 3.6 on my mac, is it best to use [Conda](https://docs.conda.io/projects/conda/en/latest/index.html)? Also which version of Conda? Looks like there are a few versions; Anaconda®, Miniconda, Anaconda Repository, Anaconda Enterprise.

And doing something [like this](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-python.html)

`conda create -n py36 python=3.6 anaconda`

seems right?

**Comment by Tmjk** [comment]
1.13.1 != 1.12.0

**Comment by gabrielsaviank** [comment]
Are you using Pycharm ? I was getting this error in pycharm, and then I realised that I was running version 3.8 in the IDE and 3.6 in the Terminal. Check in the bottom of the Pycham and set to the 3.6 and it'll work.

**Comment by GnarGnarHead** [comment]
yeah I've bumped into this same error on the Docker instillation, has got me stumped

**Comment by TBAIKamine** [comment]
according to the [official](https://www.tensorflow.org/install/pip) website of tensorflow on how to install using pip
to install tensorflow version 1.12 you are limited to use python versions between 3.6 to 3.7
I also tried python 3.7 and didnt work so your only option really is python 3.6
thus after using a venv virtual environement (I really hate complicating things and use different modules to do a simple task of creating a virtual environement) it appears that pip3 is a total waste of time
finally, using a simple `pip install tensorflow==1.12.0` worked
